datasource/local/src/main/java/com/egdroid/datasource/local/movie/DataConverter.kt

```diff
@@ -0,0 +1,30 @@
+package com.egdroid.datasource.local.movie
+
+import androidx.room.TypeConverter
+import com.google.gson.Gson
+import com.google.gson.reflect.TypeToken
+
+class DataConverter {
+
+    companion object {
```

Comment:
If we are going to use only DataConverter, we can make it like this
```suggestion
object DataConverter {
```
instead of 
```
class DataConverter {
```